internal/contentcache/contentcache.go

```diff
@@ -36,6 +40,34 @@ type ContentCache struct {
 	mtimeClock     timeutil.Clock
 }
 
+// Populates the cache with existing persisted files when gcsfuse starts
+func (c *ContentCache) PopulateCache() {
+	if c.tempDir == "" {
+		c.tempDir = "/tmp"
+	}
+	files, err := ioutil.ReadDir(c.tempDir)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
This can happen. For instance, if the disk is network attached, it's possible to lose the connection to the dir. To crash the gcsfuse process seems too aggressive to handle such a condition.

Reply:
I agree.  A better solution is to disable the content cache and print a log to the console if the cache directory cannot be read.